docs/tutorials/admin-bar-best-practices.md

```diff
@@ -0,0 +1,336 @@
+
+**How Grouped and Ungrouped Items Work Together:**
+
+When mixing groups of items and ungrouped menu items, the system:
+1. Positions all items first (based on `order` or module initialization order in the `app.js` file)
+2. Then ensures grouped items stay together, with their position determined by their first item
+
+<AposCodeBlock>
+
+```javascript
+// Group position is determined by its first item's position
+export default {
+  options: {
+    // The content group will appear between image and file modules
+    order: ['@apostrophecms/image', 'content-group-leader', '@apostrophecms/file'],
+    groups: [
+      {
+        name: 'content',
+        label: 'myproject:content',
+        // These stay together and appear where content-group-leader is positioned
+        items: ['content-group-leader', 'article', 'event']
+      }
+    ]
+  }
+};
+```
+  <template v-slot:caption>
+    modules/@apostrophecms/admin-bar/index.js
+  </template>
+</AposCodeBlock>
+
+> [!TIP]
+> Place frequently-used items at the beginning of the admin bar for quick access.
+
+## Managing the Quick Create Menu
+
+The Quick Create menu (the "+" icon) should only contain frequently created content types. Remove items to streamline the experience:
+
+<AposCodeBlock>
+
+```javascript
+export default {
+  options: {
+    // Prevent this content type from appearing in the quick create menu
+    quickCreate: false
+  }
+};
+```
+  <template v-slot:caption>
+    modules/article/index.js
+  </template>
+</AposCodeBlock>
+
+**When to remove items:**
+- Rarely created content types
+- Specialized content for specific roles only (even if only selected roles can create a specific piece-type, the menu shouldn't be over-cluttered)
+- When you have many piece types and need to reduce options
+- Content types requiring careful setup that should use the full content manager
+
+## Customizing the Context Utility Menu
+
+Add custom utilities to the right side of the admin bar from the `init` method of any module:
+
+```javascript
+export default {
+  init(self) {
+    // Add item to the context utility menu (right side of admin bar)
+    self.apos.adminBar.add(
+      'custom-module:help', // Unique id
+      'myproject:getSupport', // Label
+      false,                 // Permissions (false = available to all)
+      {
+        contextUtility: true,  // This makes it appear in the right side
+        icon: 'question-circle', // Icon to display (required)
+      }
+    );
+  }
+};
+```
+> [!NOTE]
+> Custom actions or link navigation need to be added through browser-side scripting as shown in the [Adding External Link Buttons](/tutorials/adding-admin-bar-external-links.html#handling-the-button-click-event) tutorial.
+
+> [!TIP]
+> The `icon` property requires a registered icon name. See the [Icons documentation](/reference/module-api/module-overview.html#icons) for available icons and how to register custom ones.
```

Comment:
Unrelated to the pr but noticing that Tips get a blue text. They should just be the standard gray/black

<img width="855" height="399" alt="Image" src="https://github.com/user-attachments/assets/eb156ac0-656f-4454-b2c8-550694dfff1d" />

Reply:
Fixed